spec/migrations/20250317000666_remove_ovirt.rb

```diff
@@ -0,0 +1,21 @@
+require 'spec_helper'
+
+migration '20250317000666_remove_ovirt' do
+  scenarios %w[foreman] do
```

Comment:
I think you can extend this to foreman and katello, as the latter also has that migration